update server installation docs (#1997)
mendersoftware/mender-docs: 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md

---
title: OpenSearch
taxonomy:
    category: docs
    label: tutorial
---

> The Mender server uses OpenSearch as storage and search engine.

Reviewer: ```suggestion
The Mender server uses OpenSearch as a storage and search engine.
```

Author: fixed

> To install OpenSearch on Kubernetes cluster using the Helm chart follow instructions from [Opensearch using `helm`](https://opensearch.org/docs/2.0/opensearch/install/helm/):

Reviewer: ```suggestion
To install OpenSearch on a Kubernetes cluster using the Helm chart follow instructions from [Opensearch using `helm`](https://opensearch.org/docs/2.0/opensearch/install/helm/):
```

Reviewer 2: We're using opensearch 2.4 version:
```suggestion
To install OpenSearch on a Kubernetes cluster using the Helm chart follow instructions from [Opensearch using `helm`](https://opensearch.org/docs/2.4/install-and-configure/install-opensearch/helm/):
```

Author: fixed
